io.openems.edge.evcs.goe.chargerhome/src/io/openems/edge/evcs/goe/chargerhome/GoeChargerHomeImpl.java

```diff
@@ -0,0 +1,340 @@
+				this.channel(GoeChannelId.VOLTAGE_L1).setNextValue(nrg.get(0).getAsInt());
+				this.channel(GoeChannelId.VOLTAGE_L2).setNextValue(nrg.get(1).getAsInt());
+				this.channel(GoeChannelId.VOLTAGE_L3).setNextValue(nrg.get(2).getAsInt());
+				this.channel(GoeChannelId.CURRENT_L1).setNextValue(nrg.get(4).getAsInt()*100);
+				this.channel(GoeChannelId.CURRENT_L2).setNextValue(nrg.get(5).getAsInt()*100);
+				this.channel(GoeChannelId.CURRENT_L3).setNextValue(nrg.get(6).getAsInt()*100);
+				this.channel(GoeChannelId.ACTUAL_POWER).setNextValue(nrg.get(11).getAsInt()*10);
+		
+				this.channel(Evcs.ChannelId.PHASES).setNextValue(phases);				
+				this.channel(Evcs.ChannelId.MAXIMUM_POWER).setNextValue(GetMaxPower(phases));	
+				this.channel(Evcs.ChannelId.MINIMUM_HARDWARE_POWER).setNextValue(GetMinHardwarePower(phases, cabelCurrent));	
+				this.channel(Evcs.ChannelId.MAXIMUM_HARDWARE_POWER).setNextValue(GetMaxHardwarePower(cabelCurrent));	
+				this.channel(Evcs.ChannelId.CHARGE_POWER).setNextValue(nrg.get(11).getAsInt()*10);
+				this.channel(Evcs.ChannelId.CHARGING_TYPE).setNextValue(ChargingType.AC);
+				
+				this.channel(GoeChannelId.ENERGY_TOTAL).setNextValue(json.get("eto").getAsInt()*100);				
+				this.channel(Evcs.ChannelId.ENERGY_SESSION).setNextValue(json.get("dws").getAsInt()*10/3600);
+			
+				this.channel(GoeChannelId.ERROR).setNextValue(err);
+				this.channel(Evcs.ChannelId.CHARGINGSTATION_COMMUNICATION_FAILED).setNextValue(false);
+				
+				this.setEnergySession();
+				this.setPower();				
+				
+			}
+			else{
+				this.channel(Evcs.ChannelId.CHARGINGSTATION_COMMUNICATION_FAILED).setNextValue(true);
+			}
+			
+			break;
+		}
+	}
+	
+	private int ConvertGoeStatus(int status) {
+		switch(status) {
+		case 1: // ready for charging, car unplugged
+			return Status.NOT_READY_FOR_CHARGING.getValue();
+		case 2: // charging
+			return Status.CHARGING.getValue();
+		case 3: // waiting for car
+			return Status.READY_FOR_CHARGING.getValue(); 
+		case 4: // charging finished, car plugged
+			return Status.CHARGING_FINISHED.getValue();
+		default:
+			return Status.UNDEFINED.getValue();
+		}
+	}
+	
+	private int ConvertGoePhase(int phase) {
+		int Phasen = (byte)phase & 0b00111000;			
+		switch(Phasen) {
+		case 8: // 0b00001000: Phase 1 is active
+			return 1;
+		case 24: // 0b00011000: Phase 1+2 is active
+			return 2;
+		case 56: // 0b00111000: Phase1-3 are active
+			return 3;
+		default:
+			return 0;
+		}
+	}
+	
+	private int GetMaxHardwarePower(int cableCurrent) {
+		int MaxPower = 0;
+		int phases = 3;
+		if (cableCurrent < this.MaxCurrent && cableCurrent > 0) {
+			MaxPower = phases*cableCurrent*230/1000;
+		}
+		else {
+			MaxPower = phases*this.MaxCurrent*230/1000;
+		}
+		return MaxPower;
+	}
+	
+	private int GetMinHardwarePower(int phases, int cableCurrent) {
+		int MinPower = 0;
+		int MinPowerHw = phases*this.MinCurrent*230/1000;
+		if (cableCurrent < this.MinCurrent && cableCurrent > 0) {
```

Comment:
Take always the greatest of both.
this.MinCurrent can be set to more than the 6A coming from the charger.
For example when we activate a Renault Zoe mode in the UI we changed the configured minimum to 10A because the Zoe is not charging bellow

Reply:
That's true but i meen he wantet to define the absolut minimum power that the charger can output

-> For sure it's a good idea to make this changeable

Reply:
okay, then i've to set the minimum always to this.minCurrent. Otherwise with a 32A cable the minCurrent is the same as the maxCurrent